### ISSUE TITLE
No more code-signature in v 0.9.4

### ISSUE THREAD

**Issue by suschizu** [1kc/razer-macos#590]
Hello - I've noticed that the code signature is missing in MacOS download 0.9.4 (https://github.com/1kc/razer-macos/releases/download/v0.4.9/Razer.macOS-0.4.9-universal.dmg) 
it's empty insead of Ville Karja (P55C5BCV2E)
can you confirm this? 
thanks!

**Comment by core-code** [comment]
i can confirm the issue.

**Comment by 1kc** [comment]
Yep that’s expected because I cannot signed due to the lack of a Apple Developer license
😢
